reference/help.txt

```diff
@@ -104,7 +104,7 @@ File Types (-i and -o options):
 	  ver                   Version of gdb file to generate (1..3)
 	  via                   (0/1) Drop route points that do not have an equivalent w
 	  dropwpt               (0/1) Don't create waypoints for non-user points
-	  roadbook              (0/1) Include major turn points (with description) from
```

Comment:
They come back because disp_formats, disp_filters, disp_vecs, and disp_filter_vecs produce them.  When I update the reference files I copy the exact file produced from TMPDIR to REFERENCE, and then make sure the changes are appropriate.